static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -243,7 +243,6 @@ const bootStandard = (): void => {
         });
     }
 
-    // initilaise the email/outbrain check mediator
     initCheckMediator();
```

Comment:
@shtukas here for instance `initCheckMediator` may not be needed anymore

Reply:
Done